tardis/tests/test_rpacket.py

```diff
@@ -0,0 +1,112 @@
+import os
+import random
+import inspect
+from ctypes import CDLL, c_double
+
+import pytest
+import numpy as np
+
+from tardis import __path__ as path
+
+test_path = os.path.join(path[0], 'montecarlo', 'montecarlo.so')
+
+tests = CDLL(test_path)
+
+np.random.seed(1)
+
+def generate_testcases_doubles(metafunc):
+	from sys import float_info as floats
+	MAX_FLOAT, MIN_FLOAT = floats[0], floats[3]
+	for each in metafunc.fixturenames:
+		if 'double_value' in inspect.getargspec(each):
```

Comment:
This is supposed to see if `double_value` is in any of the func's parameters, but it doesn't work.